apps/dashboard/templatetags/a4_candy_dashboard_tags.py

```diff
@@ -12,3 +12,30 @@ def is_publishable(project, project_progress):
     """
     return (project_progress['project_is_complete']
             and project.published_modules.count() >= 1)
+
+
+@register.filter
+def project_nav_is_active(dashboard_menu_project):
+    """Check if the view is in the project dashboard nav."""
```

Comment:
The docstring seems to be copied from somewhere and not changed? Or I don't get it anymore...

Reply:
Also, I think it would make sense to add these template tags to a4 as we use them here and in mB. But we can do that in a next PR.